magstock/templates/baseextra.html

```diff
@@ -0,0 +1,8 @@
+<script>
```

Comment:
This is the new API for adding links to the main admin menu at the top of the screen.  Plugins could theoretically also remove or modify existing links provided by the main repo this way.